website/content/de/egs/platforms/glossary/_index.md

```diff
@@ -0,0 +1,44 @@
+
+## Fähigkeiten der Plattform
+Die spezifischen Ergebnisse für den Benutzer, oder **_was_** eine Plattform bietet. Diese sind nicht zu verwechseln mit Plattformeigenschaften, die beschreiben, **_wie_** die Fähigkeiten funktionieren. Diese Fähigkeiten können auf verschiedenen Abstraktionsebenen liegen (z. B. eine einzelne Datenbank oder eine Testumgebung mit einer Datenbank) und von verschiedenen Anbietern bereitgestellt werden. Mit zunehmender Reife der Plattformen werden diese Anbieter im Allgemeinen bestrebt sein, Fähigkeiten über Selbstbedienung anzubieten, angefangen bei der Auffindbarkeit der verfügbaren Fähigkeiten bis hin zur konsistenten Nutzererfahrung über alle Funktionen hinweg. Die Fähigkeiten selbst sind oft recht langlebig, während sich die Anbieter und die Implementierung schneller weiterentwickeln können. Es ist zum Beispiel unwahrscheinlich, dass ein Unternehmen keine Testumgebungen mehr benötigt, aber es kann sich dahin entwickeln, dass es statt VM-basierter Lösungen containerisierte Lösungen anbietet.
+
+## Anbieter von Plattformfähigkeiten
+Eine Gruppe von Personen, die eine von der Plattform angebotene Fähigkeit entwickeln und pflegen. Bei den Anbietern kann es sich um externe Organisationen oder interne Teams handeln, und in kleineren Organisationen sind es oft dieselben Personen, die auch die breitere Plattform entwickeln. Wenn Plattformen reifen, profitieren sie von der Beibehaltung von Abstraktionen für die Anbieter, um ein „Lock-in“ zu verhindern und die Entwicklung hin zur „Thinnest Viable Platform“ voranzutreiben.
+
+## Eigenschaften der Plattform
+Bezieht sich darauf, **_wie_** die Plattform und ihre Fähigkeiten funktionieren und was in Bezug auf funktionsübergreifende Anforderungen, nicht-funktionale Anforderungen oder einfach Qualitäten erwartet werden kann. Beispiele hierfür sind die Zuverlässigkeit oder Leistung von verwalteten Diensten, die anhand von Service Level Objectives (SLOs) gemessen werden können, die Sicherheit, die anhand der Zeit bis zur Abschwächung identifizierter Risiken gemessen werden kann, oder die Beobachtbarkeit, die sowohl zur Fehlersuche als auch zur Berichterstattung über die Nutzung der Plattform verwendet werden kann. Eigenschaften werden oft mit Fähigkeiten verwechselt, da einige Konzepte wie die Beobachtbarkeit als Fähigkeit (z. B. OTel Operator, der zur Erfassung von Anwendungstelemetrie bereitgestellt wird) und als deklarierte Qualität (z. B. Plattformmetriken zur Messung und Meldung der Betriebszeit des bereitgestellten OTel Operators) angeboten werden können.
```

Comment:
"verwalteten Diensten", sollten wir hier nicht eher "betriebenen" nehmen? Beim "managed" geht es ja mehr um Betrieb als Verwaltung, oder? Sollten wir dann auch in den anderen Paragraphen nochmal prüfen, wie wir "managed" übersetzen.

Reply:
Meinst du wir sollten managed vielleicht gar nicht übersetzen? Hmm doch, würde ich schon übersetzen, aber guter Punkt, dass Betrieb in dem kontext defininitv besser passt als Verwaltung.